models/iaf_cond_alpha_mc_fixedca.h

```diff
@@ -0,0 +1,663 @@
+    double_t jump_Th;           //!< Jump in adaptive threshold upon spike in mV
+    double_t tau_Th;            //!< Time constant for adaptive threshold in ms
+    double_t t_refCa;           //!< Refractory period for calcium spike in ms
+    double_t act_flag;          //!< flag to turn on calcium spikes
+    double_t reset_flag;        //!< flag to set somatic membrane potential to reset value at end of
+    // refractory period
+
+    double_t g_conn[ NCOMP - 1 ]; //!< Conductances connecting compartments, in nS
+    double_t t_L[ NCOMP ];        //!< Leak during refractory period in nS
+    double_t nt_L[ NCOMP ];       //!< Leak at other times in ns
+
+    double_t g_L[ NCOMP ];      //!< Leak Conductance in nS
+    double_t C_m[ NCOMP ];      //!< Membrane Capacitance in pF
+    double_t E_ex[ NCOMP ];     //!< Excitatory reversal Potential in mV
+    double_t E_in[ NCOMP ];     //!< Inhibitory reversal Potential in mV
+    double_t E_L[ NCOMP ];      //!< Leak reversal Potential (aka resting potential) in mV
+    double_t tau_synE[ NCOMP ]; //!< Synaptic Time Constant Excitatory Synapse in ms
+    double_t tau_synI[ NCOMP ]; //!< Synaptic Time Constant for Inhibitory Synapse in ms
+    double_t I_e[ NCOMP ];      //!< Constant Current in pA
+    // changes
+    double_t
+      tau_curAP[ NCOMP ]; //!< Time constant of active current at each compartment after AP in pA
+    double_t amp_curAP[ NCOMP ]; //!< Amplitude of active current at each compartment after AP in pA
+
+    Parameters_();                                //!< Sets default parameter values
+    Parameters_( const Parameters_& );            //!< needed to copy C-arrays
+    Parameters_& operator=( const Parameters_& ); //!< needed to copy C-arrays
+
+    void get( DictionaryDatum& ) const; //!< Store current values in dictionary
+    void set( const DictionaryDatum& ); //!< Set values from dicitonary
+  };
+
+
+  // State variables  ------------------------------------------------------
+
+  /**
+   * State variables of the model.
+   * @note Copy constructor and assignment operator required because
+   *       of C-style array.
+   */
+public:
+  struct State_
+  {
+
+    /**
+     * Elements of state vector.
+     * For the multicompartmental case here, these are offset values.
+     * The state variables are stored in contiguous blocks for each
+     * compartment, beginning with the soma.
+     */
+    enum StateVecElems_
+    {
+      V_M = 0,
+      DG_EXC,
+      G_EXC,
+      DG_INH,
+      G_INH,
+      DI_AP,
+      I_AP,
+      STATE_VEC_COMPS
+    };
+
+    //! total size of state vector
+    static const size_t STATE_VEC_SIZE = STATE_VEC_COMPS * NCOMP;
+
+    //! neuron state, must be C-array for GSL solver
+    double_t y_[ STATE_VEC_SIZE ];
+    int_t r_; //!< number of refractory steps remaining
+    // changes
+    int_t rCa_;   //!< number of refractory steps remaining for calcium spike
+    int_t numCa_; // num of calcium spikes
```

Comment:
Should be `//!<` to make doxygen happy, also on further lines below; some members lack a comment.